### PR TITLE
Enable testing

### PR DESCRIPTION
**Pull request by marcfir**
Enables testing and removes old legacy code.

### CODE REVIEW

**Review by ccvca — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -28,6 +28,10 @@ jobs:
             -DCMAKE_BUILD_TYPE=${{matrix.build_type}} -DPAHO_WITH_SSL=1 -DBUILD_DEB_PACKAGE:BOOL=1 -DDEB_PACKAGE_LIBCPP_VERSION=$LIBCPP_VERSION
           cmake --build . -j
           make package
+      - name: Run Tests
+        run: |
+          cd build/Dashboard-Client-build
+          ctest -V
```
Comment (GoetzGoerisch) [inline .github/workflows/build.yml:34]: Why is `-C ${{matrix.build_type}}` missing here?

**Review by GoetzGoerisch — COMMENTED** [review]
Why are three tests removed?

File: .github/workflows/build.yml (hunk 1 above)
Comment (marcfir) [reply .github/workflows/build.yml]: Good point. I fixed this.

Comment (marcfir) [reply .github/workflows/build.yml]: > Why are three tests removed?
These tests are for the old Unified Automation SDK.

**Review by GoetzGoerisch — APPROVED** [review]

**Comment by GoetzGoerisch** [comment]
Merged, Thanks! 👍🏻

**Pull request merged**